src/main/resources/application.yaml

```diff
@@ -70,6 +70,17 @@ deployer:
       processedCommits:
         # The folder path where processed commit files are stored
         folderPath: ${deployer.main.homePath}/processed-commits
+      pool:
+        # Thread pool core size
+        size: 10
+        # Thread pool max size
+        max: 20
+        # Thread pool queue size
+        queue: 10
```

Comment:
Can't we have an unbounded number? If there's more a request comes and there's already 10 on queue, won't it drop the request?

Reply:
If the queue is unbound the pool size never increases, requests will be dropped when the pool reaches the max size and the queue is full too

Reply:
Yeah I'm ok with the pool size not increasing, but requests shouldn't be dropped at such a low amount. Make the queue 10000.